src/disk_utils.cpp

```diff
@@ -1107,7 +1109,9 @@ int build_disk_index(const char *dataFilePath, const char *indexFilePath, const
     std::string labels_file_original = label_file;
     std::string index_prefix_path(indexFilePath);
     std::string labels_file_to_use = index_prefix_path + "_label_formatted.txt";
```

Comment:
Should we make all the suffixes as const strings in some header file?

Reply:
we should

Reply:
> Should we make all the suffixes as const strings in some header file?

same suggestion as above :)